ui/Screen/OrgRegistration.svelte

```diff
@@ -62,6 +62,7 @@
   const registerOrg = async () => {
     try {
       await register(orgName);
+      await fetchSession();
```

Comment:
Good catch on getting the fresh session. This seems to be a common pattern throughout the app, wonder how to address this consistently. Maybe with notifications about new session state. Open question I guess.

Reply:
Yea, was wondering about how the Svelte context API will scale for us, as it is not reactive.

> Context is not inherently reactive. If you need reactive values in context then you can pass a store into context, which will be reactive.
https://svelte.dev/docs#setContext

In the future we'll want new data to be pushed to the front-end, then it would be good that the store changes actually propagate to the components automatically. Maybe we could just use global stores instead of the context API then?

Reply:
Here's more from Svelte docs:
> Contexts vs. stores
>
>Contexts and stores seem similar. They differ in that stores are available to any part of an app, while a context is only available to a component and its descendants. This can be helpful if you want to use several instances of a component without the state of one interfering with the state of the others.
>
>In fact, you might use the two together. Since context is not reactive, values that change over time should be represented as stores
>
>https://svelte.dev/tutorial/context-api

Reply:
That is actually really good insight and definitely something we need to change going forward. My point was more about taming the dependence on fetching the session fresh when an action has been submitted. We can rework the session passing into a store.